src/main/java/org/broadinstitute/hellbender/engine/FeatureManager.java

```diff
@@ -151,7 +151,29 @@ public FeatureManager(final CommandLineProgram toolInstance, final int featureQu
         this.toolInstanceSimpleClassName = toolInstance.getClass().getSimpleName();
         this.featureSources = new LinkedHashMap<>();
 
-        initializeFeatureSources(featureQueryLookahead, toolInstance, cloudPrefetchBuffer, cloudIndexPrefetchBuffer, reference);
+        initializeFeatureSources(featureQueryLookahead, toolInstance, cloudPrefetchBuffer, cloudIndexPrefetchBuffer, reference, GATKTool.ERROR_ON_OUT_OF_DATE_INDEX_DEFAULT);
```

Comment:
this is gross, I don't like it.  Maybe we can find a better way.

Reply:
Agreed.  I added it to the bottom of StandardArgumentDefinitions as StandardArgumentDefinitions.ERROR_ON_OUT_OF_DATE_INDEX_DEFAULT_VALUE.  I thought about adding in another file for the standard argument default values, but it makes sense to me to keep the defaults with the definitions.